### DISCUSSION TITLE
옆 집의 협업 훔쳐보기

### DISCUSSION THREAD

**Discussion by ccmuk58** [The-Day-In-1984/1984_Project#38] [General]
평소처럼 파도타기하다가 Ludico 팀의 KNOCKturne 레포지토리를 발견했는데,,

이슈나 템플릿, 라벨 같은 부분 참고하면 도움될 것 같습니다.

(염탐 허가받았습니다,,,! 얏호)

[Ludico_KNOCKturne](https://github.com/cedongne/Ludico_KNOCKturne)

**Comment by fkdl0048** [comment]
냠냠입니다..!
